### ISSUE TITLE
Add the division of chats into global/local and others

### ISSUE THREAD

**Issue by wundertraume** [AdvancedPlugins/Chat#106]
### Describe the feature

I did not understand how to split the chats, so I concluded that this is not possible now. Make it so that you can create your own chats and use a certain character to write in the chat, as examples:

GLOBAL CHAT
Symbol: "!"
Permission: true #advancedchat.global
Example of the message: "!Hello World"

LOCAL CHAT
Symbol: ""
Permission: true #advancedchat.local
Example message: "Hello World"

ADMIN CHAT
Symbol: "#"
Permission: true #advancedchat.local
Example message: "#Hello World"

TRADING CHAT
Symbol: "$"
Permission: true #advancedchat.local
Example message: "$Hello World"

**Comment by ThomasWega** [comment]
Yeah this is not how it's currently possible. You have one quick message characted (default is "!") and then you write the channel name after it. So like !global Hello guys

**Comment by wundertraume** [comment]
whyyyy... its so hard dude....

**Comment by Ideecker** [comment]
It would be great if you could add that to this plugin.

**Comment by ThomasWega** [comment]
Hey! Great suggestion! We have moved suggestions to discord, where community votes on features to be added. Not a lot of people use github. If you are still interested in us adding this feature, please add it in the discord server. Thank you! <3
